src/db/seeders/createSuperAdmin.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+import models from '../models';
+
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const { User } = models;
+
+const {
+  SUPER_ADMIN_STATUS,
+  SUPER_ADMIN_EMAIL,
+  SUPER_ADMIN_PASSWORD,
+  SUPER_ADMIN_USERNAME
+} = process.env;
+/**
+ * @description class representing create superUser
+ *
+ * @class CreateSuperAdmin
+ */
+class CreateSuperAdmin {
+  /**
+   * @description - This method is responsible for creating super user
+   * @static
+   * @returns {object} - object representing response message
+   * @memberof CreateSuperAdmin
+   */
+  static async registerSuperAdmin() {
+    try {
+      await User.findOrCreate({
+
+        where: { $or: [{ username: SUPER_ADMIN_USERNAME }, { email: SUPER_ADMIN_EMAIL }] },
+        defaults:
+        {
+          username: SUPER_ADMIN_USERNAME,
+          email: SUPER_ADMIN_EMAIL,
+          password: SUPER_ADMIN_PASSWORD,
+          role: SUPER_ADMIN_STATUS
+        }
+      });
+      console.log('superAdmin account created successfully');
+    } catch (error) {
+      console.log(error.message);
```

Comment:
What does the consumer of the API see when things go wrong?

Reply:
Thanks for pointing this out.

Reply:
Resolved.